Podfile

```diff
@@ -10,6 +10,7 @@ target 'Sefirah' do
 	platform :ios, '8.0'
 	pod 'KosherCocoa', :git => 'git@github.com:MosheBerman/KosherCocoa.git', :commit => '5376523'
 	pod 'CircleProgressView', :git => 'https://github.com/CardinalNow/iOS-CircleProgressView.git'
+	pod "SugarRecord/CoreData"
```

Comment:
@jesiegel1 why don't you use Pods versions in the Podfile, instead of referencing a git url and/or commit?